ignite/metrics/precision_recall.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import division
+
+import torch
+
+from ignite.metrics.accuracy import _BaseClassification
+from ignite.exceptions import NotComputableError
+from ignite._utils import to_onehot
+
+
+class _BasePrecisionRecallSupport(_BaseClassification):
```

Comment:
Maybe we can rename it to `_BasePrecisionRecall` ?
I understood that 'support' was coming from sklearn ...

Reply:
Agree!

Reply:
Incorporated.